wire/modules/AdminTheme/AdminThemeUikit/init.php

```diff
@@ -40,6 +40,13 @@
 $config->set('inputfieldColumnWidthSpacing', 0); 
 $config->js('ukGridWidths', $ukGridWidths);
 
+$roles = array();
+foreach ($this->user->roles as $role) {
+	array_push($roles, $role->name);
+}
+$config->js('roles', $roles);
```

Comment:
Why not just `$config->js('roles', $this->user->roles->name())`?

Reply:
Because it returns an array. I thought about using the roles as an additional css class on the body element. I think your code is better if you plan only for using it from the javascript side, but then we would need an additional script to add the classes to the body.

Hmmm, really don't know whats best.

Reply:
Yes, your code is much simpler